cfpb/cfgov-sheer-templates
# Add support for cfgov-refresh header and footer

This PR adds support for including the cfgov-refresh header and footer via `global_include` in the HTML templates and building the necessary CSS and JS, which is checked in in the `static/` folder.

It also removes all traces of the Nemo shim.


## Additions

- `header.html` and `footer.html` that call the on-demand versions from cfgov-refresh via `global_include`.
- CSS and JS necessary for displaying the cfgov-refresh header and footer.
- A script, `update-cfgov-refresh-stuff.sh`, for future updates of those CSS and JS files as necessary in this repo.

## Removals

- Nemo shim HTML, CSS, and JS

## Changes

- Updates `base.html` to include the new `header.html` and `footer.html` as well as the new head scripts required for this code to work.

## Testing

1. Pull this branch into a project that depends on cfgov-sheer-templates
2. Update that project's build scripts to pull in the new CSS and JS files that are being added in this PR.
3. Build
4. Serve
5. Observe

## Review

- @anselmbradford
- @rosskarchner 

## Notes

- Is `flapjack` the right branch to be pointing my cfgov-refresh dependency at?

## Todos

- [x] ~~Test it myself!~~ Working test with changes in [GHE]/CFGOV/fin-ed-resources#61

## Checklist
